extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -168,11 +164,17 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width, source, origin) {
-    this.baseInstance_.attemptChangeSize(height, width).then(
-        () => this.sendEmbedSizeResponse_(
-            true /* success */, width, height, source, origin),
-        () => this.sendEmbedSizeResponse_(
-            false /* success */, width, height, source, origin));
+    this.baseInstance_.attemptChangeSize(height, width).then(() => {
+      if (height) {
+        this.iframe_.height = height;
```

Comment:
element's width and height are been set inside `attemptChangeSize()`. And I only change iframe size when`attemptChangeSize` succeeds.